src/components/whiteButton/whiteButton.js

```diff
@@ -0,0 +1,22 @@
+const HallOfFame = document.createElement('button')
+const Icon = document.createElement('img')
+HallOfFame.textContent = "hall of fame"
+Icon.setAttribute('src', './static/assets/img/modes/icon/contacts_24px.png')
+document.body.append(HallOfFame)
+HallOfFame.append(Icon)
+
+
+HallOfFame.style.background = '#FFFFFF'
+HallOfFame.style.border = "none"
+HallOfFame.style.padding = '40px'
+HallOfFame.style.height = '100px'
+HallOfFame.style.width = '50vw'
```

Comment:
I don't recommend using `vw` unit for elements like buttons etc. because it makes them hard to manage. In most cases this unit is only useful for positioning layouts